Resources/Prototypes/Entities/Mobs/Species/base.yml

```diff
@@ -111,16 +111,26 @@
         # sprite refactor when
         state: l_leg
       - shader: StencilMask
-        map: [ "enum.HumanoidVisualLayers.StencilMask" ]
+        map: ["enum.HumanoidVisualLayers.MaleStencilMask"]
+        sprite: Mobs/Customization/masking_helpers.rsi
+        state: male_full
+        visible: false
+      - shader: StencilMask
+        map: ["enum.HumanoidVisualLayers.FemaleStencilMask"]
         sprite: Mobs/Customization/masking_helpers.rsi
         state: female_full
         visible: false
-      - map: [ "jumpsuit" ]
-      - map: [ "enum.HumanoidVisualLayers.LHand" ]
-      - map: [ "enum.HumanoidVisualLayers.RHand" ]
-      - map: [ "enum.HumanoidVisualLayers.LFoot" ]
-      - map: [ "enum.HumanoidVisualLayers.RFoot" ]
-      - map: [ "enum.HumanoidVisualLayers.Handcuffs" ]
+      - shader: StencilMask
+        map: ["enum.HumanoidVisualLayers.UnisexStencilMask"]
+        sprite: Mobs/Customization/masking_helpers.rsi
+        state: unisex_full
+        visible: false
```

Comment:
Do we need 3 different stencil masks here? Seems unnecessary (can someone really have 3 gender masks at once?)